pkg/pgmodel/ingestor/insert.go

```diff
@@ -60,7 +60,7 @@ func initializeInserterRoutine(conn pgxconn.PgxConn, metricName string, complete
 	return tableName, err
 }
 
-func runInserterRoutine(conn pgxconn.PgxConn, input chan *insertDataRequest, metricName string, completeMetricCreationSignal chan struct{}, metricTableNames cache.MetricCache, toCopiers chan copyRequest) {
+func runInserterRoutine(conn pgxconn.PgxConn, input chan *insertDataRequest, metricName string, completeMetricCreationSignal chan struct{}, metricTableNames cache.MetricCache, toCopiers chan copyRequest, labelArrayOID uint32) {
```

Comment:
(optional) This line is already too long. I think we should switch to a struct for this.

Reply:
fixed formatting; leaving change to struct later